spec/openapi.yaml

```diff
@@ -1736,6 +1736,97 @@ paths:
           curl -H "Authorization: Bearer $smartlingToken" -o $smartlingLocaleId$smartlingFileUri
           -G --data-urlencode "fileUri=$smartlingFileUri"
           "https://api.smartling.com/files-api/v2/projects/$smartlingProjectId/locales/$smartlingLocaleId/file"
+  
+  /files-api/v2/projects/{projectId}/locales/{localeIds}/file:
+    get:
+      summary: Download translated file (Multi-locales)
+      tags:
+      - Files
+      description: |
+        This downloads a multi-locale version of the requested file, where translations for all specified locales are delivered inside one combined file.
+        
+        Multi-locale download is supported only for the following file types:
+        MADCAP, DITA_ZIP, CSV, XLSX_TEMPLATE, and XCSTRINGS.
+        All other file types must be downloaded using the single-locale API.
+
+        It is important to check the HTTP response status code. If Smartling
+        finds and returns the file normally, you will receive a `200` SUCCESS
+        response. If you receive any other response status code other than
+        `200`, the requested file will not be part of the response.
+
+        When you upload a UTF-16 character encoded file, then /file/get requests
+        for that file will have a character encoding of UTF-16. All other
+        uploaded files will return with a character encoding of UTF-8.
+
+        You can always use the content-type header in the response of a file/get
+        request to determine the character encoding.
+
+        **Note**: The filename in the response header is based on the fileUri
+        associated with the requested file. In most cases, the filename is
+        exactly the fileUri provided.
+      operationId: downloadTranslatedFileMultiLocales
+
+      parameters:
+      - name: projectId
+        description: A unique project identifier. This can be found in the Smartling Dashboard under Account Settings > API.
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: localeIds
```

Comment:
I think it would be better to have a single endpoint with all parameters in one place. It would also be more convenient for tools that generate APIs/types from the schema.
Also, I don't think `localeId` makes sense in this case, since it's just a single element of `localeIds`. Let's keep only /files-api/v2/projects/{projectId}/locales/{localeIds}/file and update description for `includeOriginalStrings` that it only works for 1 element